qadence/ml_tools/train_grad.py

```diff
@@ -29,7 +31,7 @@ def train(
     loss_fn: Callable,
     device: torch_device = None,
     optimize_step: Callable = optimize_step,
-    write_tensorboard: Callable = write_tensorboard,
+    write_tracker: Callable = write_tracker,
```

Comment:
@dominikandreasseitz , seems to me like we don't need `write_tracker` as an input, because the user defines the tracking tool via `config` and the rest is handled internally in `train_grad`.